ajna/v2/modules/auctions.py

```diff
@@ -92,6 +92,12 @@ def _update_kick_locked_amount(chain, auction_uid, is_reward, locked_change):
 def process_kick_event(chain, event):
     borrower = event.data["borrower"].lower()
     bond = wad_to_decimal(event.data["bond"])
+    cwp = chain.current_wallet_position.objects.get(
+        pool_address=event.pool_address, wallet_address=borrower, in_liquidation=False
+    )
+    wp = chain.wallet_position.objects.get(
```

Comment:
Remove this get as:
1) wallet_position has MANY records per pool_addres - wallet_address (as it's basically a state on a specific block)
2) you should update wallet_positon from current_wallet_position when wallet_position gets created